src/stcal/jump/twopoint_difference.py

```diff
@@ -119,6 +125,13 @@ def find_crs(dataa, group_dq, read_noise, normal_rej_thresh,
         row2cr, col2cr = np.where(np.logical_and(ndiffs - num_unusable_groups == 2,
                                   max_ratio > two_diff_rej_thresh))
 
+        log.info(f'From highest outlier Two-point found {len(row4cr)} pixels \
+                 with at least one CR and at least four groups')
```

Comment:
I realized while doing debugging in the last few days that these log messages are incorrect or at least deceptive, in that they refer to the number of **groups**, when in fact the quote numbers are actually the number of **differences** being worked with. It's quite impossible to do any jump detection on ramps that only have 2 **groups**, but you can do it if you have 2 **differences** available. So rather than changing the wording in the messages to say "differences" instead of "groups", let's just change the numbers that are quoted (bump them all up by 1). I.e. first message is regarding **five** groups, next one is **four** groups, and last one is **three** groups.

Reply:
I noticed this and wanted to change the logging messages, but I wasn't sure if when you were comparing old version to the new version if you wanted the exact same logging to appear for consistency. If you don't need the messages to be the same, I will go ahead and scrap these and put some different more informative messages in (including logging during/after the iteration for additional CRs). What do you think about this?

Reply:
Works for me.